### ISSUE TITLE
Missing artillery_tft_fw_1.27.x_patch_6 and other files in the respository

### ISSUE THREAD

**Issue by awiltschek** [digant73/Artillery-Sidewinder-X1_releases_tft#2]
Can you please publish also the allready rolled out Patches to your repository Patch 4, 5, 6 missing ? or at least the realy latest version woudl be nice. 
And the link to your Thingiverspage is dead. 
Thnx
Alex

**Comment by AcidSubway** [comment]
yes I miss the paches too and the pages, hopefully he continues! PLEASE PLEASE
